packages/amplify_datastore/test/test_models/Blog.dart

```diff
@@ -20,121 +20,156 @@ import 'package:amplify_datastore_plugin_interface/amplify_datastore_plugin_inte
 import 'package:collection/collection.dart';
```

Comment:
Probably will need to reformat this codegen generate file to pass format check in CI. Same to other "test models".

Reply:
Thanks I've reformatted all the files to pass the format check